### PR TITLE
[frontend/backend] Add quick actions on Assignee/Participant (#4333)

### LINKED ISSUE #4333: Quick button to add Assignees and Participants
## Use case

Add a quick button near to Assignees list to add quickly assignees. (the same can be applied to Participants)

image

### PR DESCRIPTION
**Pull request by frapuks**
### Proposed changes

* Design update of Assignee/Participant chip
* Add quick button on Assignee/Participant to open a drawer to easily add a user
* Display first the current user in the dropdown list
* Display a `-` when you don't have labels
* Uniformized spacing on Assignee/Participant/Labels with other components

### Related issues

* close #4333 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

image

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/ItemParticipants.tsx [hunk 1]
```diff
@@ -1,28 +1,65 @@
...
-        >
-          {participant.name}
-        </Button>
+        <Tooltip key={participant.id} title={participant.name}>
+          <Chip
+            key={participant.id}
+            variant="outlined"
+            icon={<PersonOutline color={'primary'} />}
+            label={truncate(participant.name, 25).toUpperCase()}
+            style={{
```
Comment (Archidoit) [inline opencti-platform/opencti-front/src/components/ItemParticipants.tsx:49]: better to include the styles in a 'useStyles' constant out of the component definition

**Comment by Archidoit** [comment]
Missing a 'S' in 'Add new Participant' (to be consistant with 'Add new labels' and 'Add new attributes' : 

image

image

(maybe also the Upper case to be consistant with 'Add new labels') : 
image

Same for Assignee(s) / Participant(s) in the text field --> Assignees / Participants (to be consistant with 'Labels')

**Comment by Archidoit** [comment]
Behavior tested, working well

File: opencti-platform/opencti-front/src/components/ItemParticipants.tsx (hunk 1 above)
Comment (frapuks) [reply opencti-platform/opencti-front/src/components/ItemParticipants.tsx:49]: Ok, but above all useStyles we have added a comment wich say : Don't use for new code. I thought good practice was like this, inline style.

Comment (Archidoit) [reply opencti-platform/opencti-front/src/components/ItemParticipants.tsx:49]: mm indeed, sorry, I don't know best practices for css

**Comment by marieflorescontact** [comment]
when opening a task after creating it from RFI > Task + > got the task, there is no quick action for `Assignees`, is it normal?
{AA67B84A-19FC-455B-8084-2FE882BA0442}

**Review by marieflorescontact — APPROVED** [review]
looks good 👌

**Pull request merged**